beetmoverscript/utils.py

```diff
@@ -119,3 +119,25 @@ def get_release_props(initial_release_props_file, platform_mapping=STAGE_PLATFOR
     expanded the properties with props beetmover knows about."""
     props = load_json(initial_release_props_file)['properties']
     return update_props(props, platform_mapping)
+
+
+def alter_unpretty_contents(original_artifact_name, source, artifact_pretty_name):
+    """
+    Function to alter any unpretty-name contents from a file.
+    Args:
+        original_artifact_name: e.g. `target.test_packages.json`
+        artifact_pretty_name: e.g. `firefox-53.0a1.en-US.linux-i686.test_packages.json`
+        source: path to CoT file
+    """
+    # remove `target` keyword and obtain the filename
+    filename = original_artifact_name.split('.', 1)[1]
```

Comment:
We can take every simple name and pass it through the manifest, right?  (We may have to use `os.path.basename` to avoid using the `en-US/` subdir)  I'm wondering if that may be a better way.

Your method works, as long as the `target` name always matches the pretty name (which might not be true).  We already have a way to translate the simple to the pretty name, though.